scripts/generateEquipment.py

```diff
@@ -114,8 +115,16 @@ def main():
 
         po = v['printouts']
         item_id = getPrintoutValue(po['Item ID'])
+
+        splitted = k.rsplit('#', 1)
```

Comment:
I think this is going to get rid of ALL variants of all items, not just uncharged. You need to run this python script to get the updated equipment.json. 

Even so, I think there are more uncharged variants that can be used other than the Scythe. Combat bracelet, amulet of glory, black mask, dragonfire shield, scythe recolors, ring of shadows come to mind. There's probably others. Even if the uncharged versions have the same stats as the normal (like a glory) it probably needs to be an alias in the EquipmentAliases.ts and not completely removed.

![image](https://github.com/weirdgloop/osrs-dps-calc/assets/7608429/047d5fde-823c-42a0-a82a-ae4406211875)

Reply:
whoops, this was not meant to be included

Reply:
this was a WIP change